esphome/components/qspi_amoled/display.py

```diff
@@ -42,6 +42,13 @@
 }
 DATA_PIN_SCHEMA = pins.internal_gpio_output_pin_schema
 
+
+def validate_dimension(value):
+    if value % 2 != 0:
```

Comment:
Also should check that these values are positive - and can then simplify the validation in the schema to just `validate_dimension` rather than the `cv.All( ...`
```suggestion
    value = cv.positive_int(value)
    if value % 2 != 0:
```